{{cookiecutter.project_slug}}/src/redux/mainReducer.js

```diff
@@ -10,5 +10,5 @@ export const combinedReducers = combineReducers({
     if (state == null) state = [];
     return state;
   },
-  {% if cookiecutter.has_login_blueprint %}EmailAuth: EmailAuthReducer{% endif %}
+  {% if cookiecutter.has_login_blueprint == "True" %}EmailAuth: EmailAuthReducer{% endif %}
```

Comment:
Please remove all the fixes to `has_login_blueprint` , we can add them in other PR 👍